src/coreclr/vm/gcstress.h

```diff
@@ -72,13 +72,22 @@ namespace GCStressPolicy
     private:
         // This static controls whether GC stress may induce GCs. EEConfig::GetGCStressLevel() still
         // controls when GCs may occur.
-        static Volatile<DWORD> s_nGcStressDisabled;
-
+        static int s_nGcStressDisabled;
         bool m_bAcquired;
 
+        FORCEINLINE static void Disable()
+        { Interlocked::Increment(&InhibitHolder::s_nGcStressDisabled); }
+
+        FORCEINLINE static void Enable()
+        {
+            int newVal = Interlocked::Decrement(&InhibitHolder::s_nGcStressDisabled);
+            _ASSERTE(newVal >= 0);
+            __UNUSED(newVal);
```

Comment:
```suggestion
            int newVal;
            newVal = Interlocked::Decrement(&InhibitHolder::s_nGcStressDisabled);
            _ASSERTE(newVal >= 0);
```
Nit: This is the usual way to make compilers happy that does not require any special macros.